### PR TITLE
9282

### PR DESCRIPTION
**Pull request by sri2k1us**

### CODE REVIEW

File: react-components/src/util/table/EnhancedTableHead.js [hunk 1]
```diff
@@ -27,22 +27,23 @@ class EnhancedTableHead extends React.Component {
             order,
             orderBy,
             numSelected,
-            rowCount,
             columnData,
             selectable,
             classes,
             padding,
+            rowInPage
```
Comment (ash3rz) [inline react-components/src/util/table/EnhancedTableHead.js]: Changing this prop name might break any components currently using `rowCount` (not sure if there are any, but thought I'd mention it).  When I first read this, I thought `rowInPage` was referring to a particular row within the page, not the total number of rows in a page.  Maybe `rowsPerPage` would be clearer?

Comment (sri2k1us) [reply react-components/src/util/table/EnhancedTableHead.js]: Makes sense. I will update prop name. And yes, I am going update all places where this component being used.

Comment (sri2k1us) [reply react-components/src/util/table/EnhancedTableHead.js]: Actually, I will name it `rowsInPage` cos it is not the page size but the actual number of rows in the page. For example, you set `100` row per page from bottom pagination toolbar but the number of rows in that page be just 10. This prop refers to `10`.

Comment (ash3rz) [reply react-components/src/util/table/EnhancedTableHead.js]: Good point - `rowsInPage` sounds good

**Comment by sri2k1us** [comment]
Closing this PR. Will re-open a new one.

**Pull request closed without merging**